### PR TITLE
EIP-908: Marking as withdrawn

### PR DESCRIPTION
**Pull request by MicahZoltu**
This EIP hasn't been touched for over 2 years.  As an author on the EIP, I don't even remember what it is at this point and after reviewing it I no longer hold any opinions I may have held on the topic two years ago.  Unless @jamesray1 plans on pushing for this, I am recommending we mark it as withdrawn.

If someone later decides to pick it up again, we can always bring it back to DRAFT.

### CODE REVIEW

**Comment by MicahZoltu** [comment]
@lightclient Can we get `Withdrawn` added to the new bot.  I was hoping once it was merged I could murge this PR and #2828, but not such luck it seems.

**Comment by lightclient** [comment]
Per EIPIP meeting 14, we're going to present the new set of statuses to ACD and then implement them into the repository.

**Comment by bisoks** [comment]
Hi, I want to know how to transfer my token from Etherscan account, to
Myetherwallet, or metamask wallet? please help me, I need understanding.

Thanks

File: EIPS/eip-908.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ eip: 908
 title: Reward clients for a sustainable network
 author: James Ray (@jamesray1), Micah Zoltu (@MicahZoltu)
 discussions-to: https://ethereum-magicians.org/t/eip-908-reward-full-nodes-and-clients/241
-status: Draft
+status: Withdrawn
```
Comment (axic) [inline EIPS/eip-908.md:6]:
```suggestion
status: Abandoned
```

So that this can be merged now.

Comment (MicahZoltu) [reply EIPS/eip-908.md:6]: I suppose continuing to wait on this is just clogging things up at this point.  I capitulate.

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - Trying to change EIP 908 state from Draft to Abandoned

**Pull request merged**